### ISSUE TITLE
ESlint output no errors or warnings when directory path includes square brackets

### ISSUE THREAD

**Issue by ZiiMs** [webpack-contrib/eslint-webpack-plugin#93]
- Operating System: Windows 10
- Node Version: v14.16.0
- NPM Version: 7.6.3
- webpack Version: 5.28.0
- eslint-webpack-plugin Version: 2.5.3

### Expected Behavior
Show errors and warnings

![](https://i.imgur.com/vwEKPJU.png)

### Actual Behavior
Nothing happens

![](https://i.imgur.com/WqXxKXa.png)

### Code

[Link to code to test](https://github.com/ZiiMs/eslint-webpack-example)

### How Do We Reproduce?
The code I provided above should suffice, has two errors and a warning in src/containers/App/index.js

All you have to do is put the project in a folder that's path includes square brackets E.G

S:\Servers\CTest\txData\CFXDefault_2A8AD9.base\resources\ **_[ziim]_** \ziim-progress\progress.client\html

If I drag my project one folder down from **_[ziim]_** it has linting, drag it one folder up it breaks again. Tried it with multiple projects.

**Comment by ricardogobbosouza** [comment]
Hi @ZiiMs 
Update your [context](https://github.com/webpack-contrib/eslint-webpack-plugin#context
)

```js
new ESLintPlugin({
  context: '../'
}),
```

**Comment by ZiiMs** [comment]
> Hi @ZiiMs
> Update your [context](https://github.com/webpack-contrib/eslint-webpack-plugin#context)
> 
> ```js
> new ESLintPlugin({
>   context: '../'
> }),
> ```

Still not linting with context added.
